eudemo/eudemo/model_managers.py

```diff
@@ -48,3 +53,31 @@ def get_state(self, name: str) -> None | Snapshot:
         Get an equilibrium state from the Equilibrium manager.
         """
         return self.states.get(name, None)
+
+
+class NeutronicsManager:
+    """Manager for neutronics"""
+
+    def __init__(self, csg_reactor, results):
+        self.csg_reactor = csg_reactor
+        self.results = results
+
+    def plot(self):
```

Comment:
Also maybe a tablulate() method

Reply:
I've just added a __str__ method so that you can just print the manager. This is because I for a reason I cant remember left the results underlying tabulate as a private method.

Secondly the volume calculation stuff has not been full fed through (this is where the dict type comes from. So I've ignored that for now but it should be integrated at some point

Reply:
Umm okay, I'm not fully following.

The __str__ is fine/good but I think it is much more intuitive to call .tabulate() then print(str(...))

Why not have both?

Reply:
Or as you have it

`bluemira_print_clean(f"{reactor.neutronics}")`

could be something like bluemira_print_clean(str(self)